src/pheval_exomiser/post_process/post_process_results_format.py

```diff
@@ -61,6 +61,9 @@ def extract_pheval_gene_requirements(self) -> [PhEvalGeneResult]:
 
 
 class PhEvalVariantResultFromExomiserJsonCreator:
+    acmg_pathogenic_label = "PATHOGENIC"
+    acmg_likely_pathogenic_label = "LIKELY_PATHOGENIC"
```

Comment:
Why define these here and then use them in lines 118 & 120?

Reply:
Nico made a comment about using static variables instead of strings in the code - mostly so it will make it easier for debugging and finding stuff in the code

Reply:
Does it?